languages/go/exercises/concept/using-map/.docs/after.md

```diff
@@ -0,0 +1,35 @@
+`map` in go is a key-value type. In other programming language, you might know map as `dict` or associative array. The value of an uninitialized map is `nil`, and you can define map as follows
+
+`make(map[string]int)`
+
+or
+
+```go
+  var foo map[string]int
+  foo = make(map[string]int)
+```
+
+> Here `string` represent the key type of map, and `int` represent the element type of map. You can change them to whatever valid golang type.
+
+To store a value in a map, you can use `=` operator like
+
+`foo["bar"] = 42`
+
+`map` is indexed by its key, therefore, you are required to use different key if you want to store a different data.
```

Comment:
Don't really mind the statement. In case we keep it I suggest:
```suggestion
A `map` is indexed by its key, therefore, you are required to use a different key if you don't want to overwrite existing data.
```

Reply:
Sure. Whatever works best for Go.